go.mod

```
@@ -0,0 +1,10 @@
+module bigJson
```

пакет лучше именовать kebab-case 
```suggestion
module big-json
```
примеры у тебя внизу
ну и название не очень соответствует задаче  - этот модуль ведь не может процессить любой большой json а только определённого формата и с определённым выводом

---

не знаю как переименовать

---

в смысле какой нейминг больше подходит? 
deliveries-parcer вполне норм